src/services/services.ts

```diff
@@ -725,7 +725,7 @@ namespace ts {
 
     class SourceMapSourceObject implements SourceMapSource {
         lineMap: number[];
-        constructor (public fileName: string, public text: string, public skipTrivia?: (pos: number) => number) {}
+        constructor(public fileName: string, public text: string, public skipTrivia?: (pos: number) => number) { }

```

Comment:
Yes it is passing linter (before and after). This was done by VS Code formatting, I thought it should be pushed.